Quoridor.Model/IPlayer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Quoridor.Model
+{
+    interface IPlayer
+    {
+        Position currentCell { get; }
+        Wall[] Walls { get; }
+        Position move(Position currentCell, Position start, Position end, bool HasWall);
+        void placeWall();
+        bool hasWon();
+        Position startPosition { get; }
+        Position endPosition { get; }
```

Comment:
about endPosition:
I think it should be an array, 'cause We have 9 positions where a player can win

Reply:
+, and i think it would be better, if these fields are arrays of Cells, not Position